src/driver/main.cpp

```diff
@@ -789,6 +790,13 @@ int main(int argc, const char* argv[])
 
     auto&& m = get_commands();
     auto cmd = args.front();
+
+    if(std::experimental::string_view(argv[1]) == "ort-sha")
```

Comment:
Instead of `experimental` i think we may want do similar to `filesystem.hpp` where we check if CPP version is >17 or not and accordingly use experimental or non-experimental headers

Reply:
Yea dont use experimental, just use `cmd`.